lib/config.js

```diff
@@ -65,14 +65,14 @@ exports.load = function(prompts) {
   if (loadPromise)
     return loadPromise;
 
-  return loadPromise = Promise.resolve()
```

Comment:
Right, sorry I didn't test it with adding brackets around it. I've tested it now and the linter doesn't complain when the whole expression is sourrounded by brackets. I'll fix it and add the brackets also to the other places you've pointed out.